### DISCUSSION TITLE
How to Update the program to the Latest Release?

### DISCUSSION THREAD

**Discussion by ImArghya7** [Dineshkarthik/telegram_media_downloader#185] [Q&A]
As the new version of Telegram_Media_Downloader has come, I'm clueless about how to update the program to the latest release.

I would appreciate any help regarding this. Thank you.

**Comment by Dineshkarthik** [comment]
Hi,

If you have cloned the repo you can simply run `git pull origin master` or if you have downloaded the repo as a zipped file first time you can again download the same way the new version and keep the old config.yaml file

**Reply by ImArghya7** [reply]
```
C:\Users\imarg\telegram_media_downloader>git pull origin master
From https://github.com/Dineshkarthik/telegram_media_downloader
 * branch            master     -> FETCH_HEAD
error: Your local changes to the following files would be overwritten by merge:
        media_downloader.py
Please commit your changes or stash them before you merge.
Aborting
Updating b2688c0..c4b8570

C:\Users\imarg\telegram_media_downloader>
```
Got this. What should I do next?

**Reply by Dineshkarthik** [reply]
you made some changes to the `media_downloader.py` file on your own either you need to commit it or undo the changes and pull the new changes from the repo again. Please read into some basic of git.

**Reply by ImArghya7** [reply]
> Please read into some basic of git.

My bad, I will surely learn some basics first 😅. Thank you for the help.
